client/src/components/Home/index.js

```diff
@@ -1,17 +1,9 @@
-import React, { Component } from 'react'
-import { Link } from 'react-router-dom'
-import SignIn from '../SignIn'
-import M from 'materialize-css'
-import './style.css'
+import React from 'react';
+import SignIn from '../SignIn';
+import './style.css';
 
 class Home extends React.Component {
-  constructor(props) {
-    super(props);
-  }
-  componentDidMount() {
-    M.AutoInit();
```

Comment:
need this whole block too otherwise the scrolling nav where it goes from one slide to the next only works the first time you're on the page then stops. (by whole block I mean lines 8 - 13.

Reply:
nope, doesn't appear to affect anything via local or heroku app.